### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by huzaus**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,4 +38,5 @@ task openSampleMac(type: Exec, dependsOn: [copySample]) {
 
 test {
     systemProperty 'jmeter.home', jmeterHome
+    systemProperty 'jmeter.properties', jmeterProperties
```
Comment (aplotnikov) [inline build.gradle:41]: Why do we need this property?

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Customize the test machine
+machine:
+
```
Comment (aplotnikov) [inline circle.yml]: I believe you don't need this empty line

File: gradleSetup.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (aplotnikov) [inline gradleSetup.sh]: I would it is not gradle setup it is more prepare jmeter properties.
When I read this file name I was thinking about installation of gradle.

File: jmeterSetup.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+if [ ! -d "$JMETER_HOME" ]; then
+    echo "Installing jmeter..."
+    cd $HOME
+    echo "Downloading jmeter..."
+    wget -c http://ftp.ps.pl/pub/apache/jmeter/binaries/apache-jmeter-3.1.tgz # download jmeter
+    echo "Extracting jmeter..."
+    tar -xf apache-jmeter-3.1.tgz # unpack jmeter
+fi
+
+
```
Comment (aplotnikov) [inline jmeterSetup.sh]: empty line

File: gradleSetup.sh (hunk 3 above)
Comment (aplotnikov) [reply gradleSetup.sh]: Maybe prepare gradle properties or something like this...

**Review by aplotnikov — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (huzaus) [reply build.gradle:41]: Because it is required for tests see JmeterAbstractGuiSpec

**Pull request merged**